src/models/schema/lamdaResponse.ts

```diff
@@ -0,0 +1,60 @@
+import { APIGatewayProxyResult } from "aws-lambda";
+
+export default class Response<T> {
+    _statusCode: number;
+    _data?: T[] = new Array(); // This is an array, in order to use scan and get whit the same response class
+    _error?: string;
+
+    constructor(statusCode: number, data?: T, error?: string) {
+        this._statusCode = statusCode;
```

Comment:
I'd do it like:
this._statusCode = data ?  200 : 500;
whitout passing it as a parameter, to avoid inconsistencies

Reply:
Ok, I'll fix this afternoon